src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -92,8 +92,8 @@ def items
         [
           system_items,
           *graph_items,
-          item_for("summary", _("Installation Summary"), icon: Icons::SUMMARY),
-          item_for("settings", _("Settings"), icon: Icons::SETTINGS)
+          item_for("summary", _("Installation Summary"), icon: Icons::SUMMARY)
+          # item_for("settings", _("Settings"), icon: Icons::SETTINGS)
```

Comment:
I'd add a note for each commented line, something like `# Not implemented yet` just to have a quick idea why it is commented out.

Reply:
ok